### PR TITLE
Models/service

### PR DESCRIPTION
**Pull request by wesleydelacruz**
##The Story
We started first by creating the different models. I was tasked to create the Service model which will serve as the main product for all transactions to be processed in the platform. This PR includes the following:

- [x]   Service model validations
- [x]   Service model rspec testing

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,3 +41,5 @@ yarn-debug.log*
 .yarn-integrity
 
 config/database.yml
+
+.tool-versions
```
Comment (analyn-cajocson) [inline .gitignore]: remove `.tool-versions`. we need this one to help other contributors for setting up this repository

File: config/database.yml.sample [hunk 2]
```diff
@@ -16,4 +16,4 @@ test:
 
 production:
   <<: *default
-  detabase: rails_project_production
+  detabase: rails_project_production
```
Comment (analyn-cajocson) [inline config/database.yml.sample]: do not commit this file

File: config/webpacker.yml [hunk 3]
```diff
@@ -53,7 +53,7 @@ development:
   compile: true
 
   # Verifies that correct packages and versions are installed by inspecting package.json, yarn.lock, and node_modules
-  check_yarn_integrity: true
+  check_yarn_integrity: false
```
Comment (analyn-cajocson) [inline config/webpacker.yml]: you don't have to commit this change

File: .gitignore (hunk 1 above)
Comment (wesleydelacruz) [reply .gitignore]: @Weltinio @rjmssevilla

File: config/webpacker.yml (hunk 3 above)
Comment (wesleydelacruz) [reply config/webpacker.yml]: ma'am paano maguuncommit?

File: config/database.yml.sample (hunk 2 above)
Comment (wesleydelacruz) [reply config/database.yml.sample]: ma'am kaya ko po cinommit kasi dinelete ko po siya kahapon, paano po siya maccheck ng github?

**Review by Weltinio — APPROVED** [review]

**Review by Weltinio — APPROVED** [review]

**Pull request merged**